starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,43 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <title>Apple Pie Recipe</title>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="./css/main.css">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,500" rel="stylesheet">
+	<title>Apple Pie Receipt</title>
 </head>
 <body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
+  <div class="picture">
+      <h1 class="recipe-title">Apple Pies</h1>
+    </div>
+  <p class="intro">This was my grandmother's apple pie recipe. I have never seen another one
+  quite like it. It will always be my favorite and has won me several first place
+  prizes in local competitions. I hope it becomes one of your favorites as well!</p>
+  <img class="recipe-info-pic" src="./images/recipe-info.png" alt="Recipe Info"><img>
+  <h2>Ingredients</h2>
+  <p>-1 recipe pastry for a 9 inch double crust pie</p>
+  <p>-1/2 cup unsalted butter</p>
+  <p>-3 tablespoons all-purpose flour</p>
+  <p>-1/4 cup water</p>
+  <p>-3 tablespoons all-purpose flour</p>
+  <p>-1/4 cup water</p>
+  <p>-1/2 cup white sugar</p>
+  <p>-1/2 cup packed brown sugar</p>
+  <p class="space">-8 Granny Smith apples - peeled, cored and sliced</p>
+  <img class="recipe-info-pic" src="./images/cooking-info.png" alt="Recipe Info"><img>
 
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
-
-Directions
-Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
+<h2>Directions</h2>
+<p>1) Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
```

Comment:
Just in case you are curious, there is a way to set your own counter with CSS, nothing to worry about, anyways.